Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/AssetBrowserBus.h

```diff
@@ -337,9 +337,12 @@ namespace AzToolsFramework
         public:
             static const AZ::EBusHandlerPolicy HandlerPolicy = AZ::EBusHandlerPolicy::Multiple;
 
-            //! Updates the asset preview panel with data about the passed entry.
+            //! Updates the asset browser inspector panel with data about the passed entry.
             //! Clears the panel if nullptr is passed
             virtual void PreviewAsset(const AzToolsFramework::AssetBrowser::AssetBrowserEntry* selectedEntry) = 0;
+
+            //! Clears the asset browser inspector panel if it has data in it
+            virtual void ClearPreview(){};
```

Comment:
Can it be a pure virtual function? Also, I think the comment is not accurate here. "if it has data in it" can be removed.

Reply:
Thanks, fixed the comment! The `QComponentEntityEditorInspectorWindow` instantiates this class, and to be a pure virtual function the `ClearPreview()` method would have to be implemented in that class. It doesn't need to be used there, so I made it a virtual function instead.